### ISSUE TITLE
Display Youtube username in profile page

### ISSUE THREAD

**Issue by waldyrious** [liberapay/liberapay.com#748]
Currently the connected accounts in the profile page show the username followed by the real name in parenthesis:

![screenshot from 2017-10-18 10-44-14](https://user-images.githubusercontent.com/478237/31711882-7060b112-b3f1-11e7-9a08-5089676571ac.png)

For YouTube, however, only the real name is shown, even though a username is defined (e.g. in the URL, which for my case is https://www.youtube.com/user/waldyrious).

Assuming this information is indeed exposed by YouTube when the connection request is made, it would be nice / more consistent to display it the same was as is done for the other accounts.

**Comment by Changaco** [comment]
I think Youtube usernames are vestigial, they've been replaced by channel IDs and names.

**Comment by waldyrious** [comment]
@Changaco indeed. I mentioned it more as a matter of consistency, especially since real/display names are shown in parenthesis for the other accounts.
